src/main/java/de/doubleslash/usb_led_matrix/view/ConfigurationView.java

```diff
@@ -430,9 +432,26 @@ void refreshButton() {
       }
    }
 
-   void turnOffAutomaticallyIfNeeded(final LocalTime currentTime) {
+   void turnOffAutomaticallyIfNeeded(final LocalDateTime currentLocalDateTime) {
       final LocalTime timeLightsTurnedOff = timeLightsTurnedOffNow.plusMinutes(CommandLineOptions.getTimeout());
-      if (!currentTime.isBefore(timeLightsTurnedOff) && !timedOut) {
+
```

Comment:
this logic seems very complicated. I expected basically the same logic than before but using LocalDateTime instead of LocalTime. Do we need all this additional logic?

Reply:
Done.